src/main/webapp/app/overview/course-exams/course-exam-detail/course-exam-detail.component.html

```diff
@@ -1,15 +1,88 @@
 <div *ngIf="exam" (click)="openExam()" id="exam-{{ exam.id }}">
-    <div class="row card-header text-white bg-secondary">
-        <h4 class="card-title text-center">{{ exam.title }}</h4>
+    <!-- Signature colors for the header: Blue for TestExams and Green for RedExams -->
+    <div [class]="'row card-header ' + (maxAttemptsReached ? '' : ' clickable ') + (exam.testExam ? 'bg-primary' : 'bg-success')">
```

Comment:
Using an object for conditional classes is a lot cleaner, despite a few chars longer in this case
```suggestion
    <div [ngClass]="{ 'row card-header': true, 'clickable': !maxAttemptsReached, 'bg-primary': exam.testExam, 'bg-success': !exam.testExam }">
```